geonames/challenges/default.json

```diff
@@ -270,7 +270,7 @@
             "operation-type": "create-index",
             "settings": {
               "index.refresh_interval": "30s",
-              "index.number_of_shards": 6,
+              "index.number_of_shards": {{number_of_shards | default(6)}},
```

Comment:
I think we should also allow overriding at least `number_of_replicas` as well.

Reply:
Also there can still be an issue if someone tries to override everything using `index_settings`, but I guess we don't want to deal with this in this PR.

Reply:
No, overriding `number_of_replicas` actually works because this definition is merged with the original one. I've pushed a commit that mentions that overriding `index_settings` is unsupported for this challenge.

Reply:
Ah thanks!